src/robot_dart/control/pd_control.hpp

```diff
@@ -7,6 +7,8 @@
 #include <robot_dart/control/robot_control.hpp>
 #include <robot_dart/robot.hpp>
 
+#include <dart/dynamics/RevoluteJoint.hpp>
```

Comment:
This should go into the cpp file to not pollute the workspace of people using robot_dart.